pylint_odoo/misc.py

```diff
@@ -213,3 +214,17 @@ def get_xml_record_ids(self, xml_file, module=None):
             xml_ids.append(
                 xml_module + '.' + xml_id + '.' + noupdate)
         return xml_ids
+
+    def get_field_csv(self, csv_file, field='id'):
+        '''Get xml ids from csv file
+        :param csv_file: Path of file csv
+        :param field: Field to search
+        :return: List of string with field rows
+        '''
+        csv_ids = []
+        with open(csv_file, 'rb') as csvfile:
+            lines = csv.DictReader(csvfile)
+            for line in lines:
+                if field in line.keys():
```

Comment:
Use
`if line.has_key(field):`

Or use a list comprehensions:
`return [line[field] for line in lines if line.has_key(field)]`